src/utils/component.js

```diff
@@ -23,3 +25,16 @@ export function mergeProps(props, defaultProps, inheritedProps, validations) {
     ...filterValidProps(props, validations)
   };
 }
+
+export function areOptionsValid(options) {
+  var isValid = true;
+
+  for (const option of options) {
```

Comment:
Let's check first that `options` is an array. Otherwise, it would throw.

Reply:
I noticed that too when I was adding in some tests ;) Have fixed it up now.